bootstrap/templates/kubernetes/apps/openebs-system/openebs/app/helmrelease.yaml.j2

```diff
@@ -21,14 +21,23 @@ spec:
     remediation:
       retries: 3
   values:
-    ndm:
-      enabled: false
-    ndmOperator:
-      enabled: false
-    localprovisioner:
-      enabled: true
-      deviceClass:
-        enabled: false
+    engines:
+      local:
+        lvm:
+          enabled: false
+        zfs:
+          enabled: false
+      replicated:
+        mayastor:
+          enabled: false
+    openebs-crds:
+      csi:
+        volumeSnapshots:
+          enabled: false
```

Comment:
Just a note: this will disable the creation of the CRDs for `VolumeSnapshot`, `VolumeSnapshotContent` and `VolumeSnapshotClass` since most people will deploy these via the `snapshot-controller` helm chart.

The 3.x branch of the openebs chart didn't provide these CRDs, so this essentially just replicates the old behaviour.